demos/object_detection_demo/cpp/main.cpp

```diff
@@ -223,7 +223,10 @@ cv::Mat renderDetectionData(const DetectionResult& result, const ColorPalette& p
         throw std::invalid_argument("Renderer: metadata is null");
     }
 
-    auto outputImg = result.metaData->asRef<ImageMetaData>().img;
+    auto& imgMetaData = result.metaData->asRef<ImageMetaData>();
+
+    cv::Mat outputImg;
+    outputImg = result.metaData->asRef<ImageMetaData>().img;
```

Comment:
Why is it changed?

Reply:
Frankly, this is patch splitting artefact, so I can revert it. But it makes code more readable, maybe we leave it ?

Reply:
How does extra variable make code more readable? Its name is just an abbreviation of type used in cast.

And now `imgMetaData` is not used actually.

Reply:
Speaking of `outputImg`, constructing empty object and then copying actual value into it also doesn't seem to be more readable than just copy construction.

Reply:
OK, I'll change it back

Reply:
Done

Reply:
The same is about `ImageModel::preprocess()`.

Reply:
done